philippa-choong/week5/mon/game/app/controllers/magic_controller.rb

```diff
@@ -0,0 +1,48 @@
+class MagicController < ApplicationController
+
+    def welcome
+    end
+
+    def search_form
+    end
+
+    def magic_results
+
+        #Create an array of 20 messages for magic 8 ball script to randomly choose from
+        
```

Comment:
You can print out the question the user has typed. Something like below:
`@question = params[:question]`
and then use this variable and print it on your magic 8 ball page